### PR TITLE
".com" buttons for all iPads

### PR DESCRIPTION
**Pull request by graphemecluster**
There *are* ".com" keys for all types of iPad, not only 12.9′.
This PR replaces the keyboard type switching key with the ".com" key in a search field.
The rime 分 key is also made visible even in search fields.

### CODE REVIEW

File: CantoboardFramework/Keyboard/InputController.swift [hunk 1]
```diff
@@ -802,10 +802,10 @@ class InputController: NSObject {
     private func refreshKeyboardContextualType() {
         guard let textDocumentProxy = textDocumentProxy else { return }
         
-        if textDocumentProxy.keyboardType == .some(.URL) || textDocumentProxy.keyboardType == .some(.webSearch) {
-            state.keyboardContextualType = .url
-        } else if inputEngine.composition?.text != nil {
+        if inputEngine.composition?.text != nil {
```
Comment (alex-the-man) [inline CantoboardFramework/Keyboard/InputController.swift:805]: Could you revert the change that put .rime having precedence over .url? When user is typing a url, they are more likely to use slash, .com, etc over typing the rime separator.

Comment (graphemecluster) [reply CantoboardFramework/Keyboard/InputController.swift:805]: But we need a way to type separator. User may type Chinese to the field for searching.
Maybe a `.rimeUrl` key should be added then.

Comment (alex-the-man) [reply CantoboardFramework/Keyboard/InputController.swift:805]: The url popout provides the separator. And I hid the separator in url popout if the input engine isn't composing in this commit: https://github.com/Cantoboard/Cantoboard/commit/4014dcb49e29b1f811f2fb33a293f4ec9b273789

Comment (graphemecluster) [reply CantoboardFramework/Keyboard/InputController.swift:805]: OK then, I’ll revert it.
